src/components/views/auth/SignUp/SignUp.js

```diff
@@ -0,0 +1,150 @@
+      <div className="container-body">
+        <LeftDiv signinText={signupText} />
+        <div className="right-block">
+          <form
+            onSubmit={e => registerUser(e)}
+            className="main-form-signup"
+          >
+            <div className="flex-input">
+              <div className="flex-field">
+                {/* <label htmlFor="firstName">First Name</label> */}
+                <div className="input-field">
+                  <InputField
+                    type="text"
+                    value={firstName}
+                    change={e => updateFirstName(e.target.value)}
+                    name="firstName"
+                    placeholder="First Name"
+                    className="form-input"
+                    required
+                  />
+                </div>
+              </div>
+              <div className="flex-field">
+                {/* <label htmlFor="lasttName">Last Name</label> */}
+                <div className="input-field">
+                  <InputField
+                    type="text"
+                    value={lastName}
+                    change={e => updateLastName(e.target.value)}
+                    name="lastName"
+                    placeholder="Last Name"
+                    className="form-input"
+                    required
+                  />
+                </div>
+              </div>
+            </div>
+            {/* <label htmlFor="email">Email</label> */}
+            <div className="input-field">
+              <InputField
+                type="email"
+                value={email}
+                change={e => updateEmail(e.target.value)}
+                name="email"
+                placeholder="Email"
+                className="form-input"
+                required
+              />
+            </div>
+            {/* <label htmlFor="password">Password</label> */}
+            <div className="input-field">
+              <InputField
+                type="password"
+                value={password}
+                change={e => updatePassword(e.target.value)}
+                name="password"
+                placeholder="Password"
+                className="form-input"
+                required
+              />
+            </div>
+
+            {/* <label htmlFor="confirmPassword">Confirm Password</label> */}
+            <div className="input-field">
+              <InputField
+                type="password"
+                value={confirmPassword}
+                change={e => updateConfirmPassword(e.target.value)}
+                name="confirmPassword"
+                placeholder="Confirm Password"
+                className="form-input"
+                required
+              />
+            </div>
+            <div className="input-field">
+              <button className="btn--block bttn-primary btn-submit" type="submit" value="Sign up"> Create your free account<i className="arrow-forward material-icons">arrow_forward</i></button>
```

Comment:
I don't know if sign up currently works but I got this error when I tried:

<img width="1440" alt="Screenshot 2019-08-30 at 12 49 25 AM" src="https://user-images.githubusercontent.com/15054956/63984017-3d73aa00-cac1-11e9-8943-1d4ef9fb2fd8.png">

Also, consider disabling the button when a signup request is still loading.